lib/byebug/commands/skip.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "byebug/command"
+require "byebug/helpers/parse"
+
+module Byebug
+  #
+  # Allows the user to continue execution until the next breakpoint, if
+  # the breakpoint file or line be different, stop again
```

Comment:
This comment reads a bit strange to me, maybe say

```ruby
  # Allows the user to continue execution until the next breakpoint, as
  # long as it is different from the current one
```

Reply:
Changed